common/ood-setup.sh

```diff
@@ -11,7 +11,9 @@ fi
 # Add user to system and apache basic auth
 groupadd ood
 useradd --create-home --gid ood ood
-echo -n "ood" | passwd --stdin ood
+passwd --delete root
+passwd --delete ood
+usermod -a -G wheel ood
```

Comment:
Why these changes?

Reply:
Got rid of removing root's password. Still trying to figure out Git reviews, so I removed the comments from before, but essentially the ask for a nil password and ood being a part of the weel group is this:  

ood (uid 1000) is the uid that starts the container. But it needs to be root for all sorts or things like booting httpd and interacting with all the httpd related file systems, and actually forking process' in the nginx stage. So ood needs to be wheel so that it can do all of that. I remove the password so it can freely do all of that in scripts. 

I do all of this so that I can be uid 1000 inside and out of the container. 

Again, I'm fine to just remove the podman stack and keep this file as is. It is used in packer image creation, where this would be problematic, so maybe this change would have to be coupled with (deferred to) changing how packer image gets created?

Reply:
This script is also used by docker image builds and vagrant too so I'd prefer to leave it as-is and if the changes are needed for podman we either drop podman or do something specific to podman outside common.